docs/guide/multiple_db_config.md

```
@@ -100,6 +100,53 @@ Select a database from the list to switch connections.
+```yaml
+builtins:
+  tools:
+    llm_connection_selection: true
+```
+
+When enabled, the LLM has access to two additional tools:
+
+- `list_database_connections`: Lists databases available for switching
+- `select_database_connection`: Switches to a specified database
+
+**Excluding Databases from LLM Switching**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)